### PR TITLE
Converts ê-*yin to ê-*yan

### PR DESCRIPTION
**Pull request by aaronfay**
Using the following search `(ê-[a-zâêîô\-]*)yin\b` replace `$1yan`.

### CODE REVIEW

File: source/PCT/pct14.xml [hunk 1]
```diff
@@ -314,7 +314,7 @@
 <w canon="mîkiwâhp" class="ni" gcat="0sNI" sgloss="lodge" stem="mîkiwâhp" suffix="0">mîkiwâhp</w>
 ,
 <w canon="êkâ" class="ipc" gcat="IPC" sgloss="not_to" stem="êkâ">êkâ</w>
-<w canon="kâ-isi-pakisâpiyin" class="vai" gcat="2sAIcnj" preverbs="kâ|cnj isi|thus" sgloss="quit_looking" stem="pakisâpi" suffix="-yan">k-êsi-pakisâpiyin</w>
```
Comment (bengodden) [inline source/PCT/pct14.xml:317]: This VAI form isn't in the dictionary but the VTI and VTA forms of /pakisāp-/ are in the dictionary. Wierd.

File: source/SSSC/sssc20.xml [hunk 2]
```diff
@@ -4035,7 +4035,7 @@
 <w canon="nisîmak" class="nda" gcat="3pNDAof1s" prefix="1" sgloss="younger_sib" stem="sîm" suffix="-ak">nisîmak</w>
 <w canon="êkâ" class="ipc" gcat="IPC" sgloss="not_to" stem="êkâ">êkâ</w>
 <w canon="kwayask" class="ipc" gcat="IPC" sgloss="properly" stem="kwayask">kwayask</w>
-<w canon="ê-isiwêpinamawiyin" class="vta" gcat="2s1sTAcnj" preverbs="ê|cnj isi|thus" sgloss="throw_on" stem="wêpinamaw" suffix="-iyan">êh-isiwêpinamawiyin</w>
```
Comment (bengodden) [inline source/SSSC/sssc20.xml:4038]: This should be hyphenated ē-isi-wēpinamawiyan

File: source/SSSC/sssc21.xml [hunk 3]
```diff
@@ -3867,7 +3867,7 @@
 <q>
 <w canon="êkosi" class="ipc" gcat="IPC" sgloss="thus" stem="êkosi">êkos</w>
 <w canon="ani" class="ipc" gcat="IPC" sgloss="then" stem="ani">ânih</w>
-<w canon="ê-pmâtisiyin" class="vai" gcat="2sAIcnj" preverbs="ê|cnj" sgloss="live" stem="pimâtisi" suffix="-yan">ê-pmâtisiyin</w>
```
Comment (bengodden) [inline source/SSSC/sssc21.xml:3870]: This should be ē-pimātisiyan. It is missing the first i in pim

File: source/SSSC/sssc22.xml [hunk 4]
```diff
@@ -1708,7 +1708,7 @@
 <w canon="êyahâ" class="interj" gcat="INTERJ" sgloss="oho" stem="êyahâ">êyahâ</w>
 !
 <w canon="tânisi" class="ipc" gcat="IPC" sgloss="how" stem="tânisi">tânisih</w>
-<w canon="kâ-ôh-pêtôwatâmikawiyin" class="vta" gcat="x2sTAcnj" preverbs="kâ|cnj ôh|thence" sgloss="make_bring_a_load" stem="pêtôwatâm" suffix="-ikawiyan">k-ôh-pêtôwatâmikawiyin</w>
```
Comment (bengodden) [inline source/SSSC/sssc22.xml:1711]: This should be kā-oh-pētowatāmikawiyan (short o in oh(ci)- and pētowatēw

File: source/SSSC/sssc24.xml [hunk 5]
```diff
@@ -4842,7 +4842,7 @@
 ,
 <w canon="ê-itêyihtaman" class="vti" gcat="2sTIcnj" preverbs="ê|cnj" sgloss="think_of" stem="itêyiht" suffix="-aman">êh-têyihtaman</w>
 ,
-<w canon="kâ-ôh-pê-wîkimiyan" class="vta" gcat="2s1sTAcnj" preverbs="kâ|cnj ôh|thence pê|thence" sgloss="live_with" stem="wîkim" suffix="-iyan">k-ôh-pê-wîkimiyin</w>
+<w canon="kâ-ôh-pê-wîkimiyan" class="vta" gcat="2s1sTAcnj" preverbs="kâ|cnj ôh|thence pê|thence" sgloss="live_with" stem="wîkim" suffix="-iyan">k-ôh-pê-wîkimiyan</w>
```
Comment (bengodden) [inline source/SSSC/sssc24.xml]: Here again the ō in ōh(ci) preverb is long? Perhaps this warrants considering another regex replacement.....

File: source/SSSC/sssc26.xml [hunk 6]
```diff
@@ -2411,7 +2411,7 @@
 <w canon="mâ" class="interj" gcat="INTERJ" sgloss="yes" stem="mâ">m</w>
 <w canon="ani" class="ipc" gcat="IPC" sgloss="then" stem="ani">âni</w>
 <w canon="âta" class="ipc" gcat="IPC" sgloss="although" stem="âta">âtah</w>
-<w canon="ê-manitôwakêyimoyin" class="vai" gcat="2sAIcnj" preverbs="ê|cnj" sgloss="think_self_powered" stem="manitôwakêyimo" suffix="-yan">êh-manitôwakêyimoyin</w>
```
Comment (bengodden) [inline source/SSSC/sssc26.xml:2414]: Should be manitowakēyimoyan (both o are short)

File: source/SSSC/sssc26.xml [hunk 7]
```diff
@@ -2411,7 +2411,7 @@
...
+<w canon="ê-manitôwakêyimoyan" class="vai" gcat="2sAIcnj" preverbs="ê|cnj" sgloss="think_self_powered" stem="manitôwakêyimo" suffix="-yan">êh-manitôwakêyimoyin</w>
```
Comment (bengodden) [inline source/SSSC/sssc26.xml]: Should be manitowakēyimoyan (both o are short)

File: source/SSSC/sssc27.xml [hunk 8]
```diff
@@ -710,7 +710,7 @@
 <w canon="wâkayôs" class="na" gcat="3sNA" sgloss="bear" stem="wâkayôs" suffix="0">wâkayôs</w>
 <w canon="ê-kî-kimotamâsk" class="vta" gcat="3s2sTAcnj" preverbs="ê|cnj kî|PAST" sgloss="steal_from" stem="kimotamaw" suffix="-isk">ê-kîh-kimotamâsk</w>
 ,
-<w canon="kâ-ôh-kî-wanihiyin" class="vta" gcat="2s1sTAcnj" preverbs="kâ|cnj ôh|thence kî|PAST" sgloss="lose" stem="wanih" suffix="-iyan">k-ô-kih-wanihiyin</w>
```
Comment (bengodden) [inline source/SSSC/sssc27.xml:713]: Here again the o in preverb -oh(ci)- is long...it should be short.

File: source/SSSC/sssc30.xml [hunk 9]
```diff
@@ -3875,7 +3875,7 @@
 <q>
 <w canon="nâpêw" class="na" gcat="3sNA" sgloss="man" stem="nâpêw" suffix="0">nâpêw</w>
 </q>
-<w canon="ê-pakosihtâyin" class="vai" gcat="2sAIcnj" preverbs="ê|cnj" sgloss="beg_for" stem="pakosihtâ" suffix="-yan">ê-pakosîhtâyin</w>
```
Comment (bengodden) [inline source/SSSC/sssc30.xml:3878]: Should be ē-pakosīhtāyan (long i)

File: source/SSSC/sssc34.xml [hunk 10]
```diff
@@ -3691,7 +3691,7 @@
 
 <seg n="133.2">
 <q rend="PRE 0 POST0">
-<w canon="ê-kâtawiyin" class="vta" gcat="2s1sTAcnj" preverbs="ê|cnj" sgloss="hide_it_from" stem="kâtaw" suffix="-iyan">êh-kâtwawiyin</w>
```
Comment (bengodden) [inline source/SSSC/sssc34.xml:3694]: This form isn't in the dictionary but makes sense

File: source/SSSC/sssc34.xml [hunk 11]
```diff
@@ -4206,7 +4206,7 @@
 <w canon="ka-nipahitin" class="vta" gcat="1s2sTA" preverbs="ka|will" sgloss="kill" stem="nipah" suffix="-itin">ka-nipahitin</w>
 ,
 <w canon="êkâ" class="ipc" gcat="IPC" sgloss="not_to" stem="êkâ">êkâ</w>
-<w canon="kâ-wî-kâ-kitosiyan" class="vta" gcat="2s1sTAcnj" preverbs="kâ|cnj wî|will dupH|H.REDUP" sgloss="address" stem="kitot" suffix="-iyan">kâ-wê-kâ-kitosiyin</w>
```
Comment (bengodden) [inline source/SSSC/sssc34.xml:4209]: This I believe should be (ēkā) kāwi kā-kitosiyan (you will not address me back)

File: source/SSSC/sssc20.xml [hunk 12]
```diff
@@ -4035,7 +4035,7 @@
...
+<w canon="ê-isiwêpinamawiyan" class="vta" gcat="2s1sTAcnj" preverbs="ê|cnj isi|thus" sgloss="throw_on" stem="wêpinamaw" suffix="-iyan">êh-isiwêpinamawiyin</w>
```
Comment (aaronfay) [inline source/SSSC/sssc20.xml]:
```suggestion
<w canon="ê-isi-wêpinamawiyan" class="vta" gcat="2s1sTAcnj" preverbs="ê|cnj isi|thus" sgloss="throw_on" stem="wêpinamaw" suffix="-iyan">êh-isiwêpinamawiyin</w>
```

File: source/SSSC/sssc21.xml [hunk 13]
```diff
@@ -3867,7 +3867,7 @@
...
+<w canon="ê-pmâtisiyan" class="vai" gcat="2sAIcnj" preverbs="ê|cnj" sgloss="live" stem="pimâtisi" suffix="-yan">ê-pmâtisiyan</w>
```
Comment (aaronfay) [inline source/SSSC/sssc21.xml]:
```suggestion
<w canon="ê-pimâtisiyan" class="vai" gcat="2sAIcnj" preverbs="ê|cnj" sgloss="live" stem="pimâtisi" suffix="-yan">ê-pmâtisiyan</w>
```

File: source/SSSC/sssc22.xml [hunk 14]
```diff
@@ -1708,7 +1708,7 @@
...
+<w canon="kâ-ôh-pêtôwatâmikawiyan" class="vta" gcat="x2sTAcnj" preverbs="kâ|cnj ôh|thence" sgloss="make_bring_a_load" stem="pêtôwatâm" suffix="-ikawiyan">k-ôh-pêtôwatâmikawiyin</w>
```
Comment (aaronfay) [inline source/SSSC/sssc22.xml]:
```suggestion
<w canon="kâ-oh-pêtôwatâmikawiyan" class="vta" gcat="x2sTAcnj" preverbs="kâ|cnj ôh|thence" sgloss="make_bring_a_load" stem="pêtôwatâm" suffix="-ikawiyan">k-ôh-pêtôwatâmikawiyin</w>
```

File: source/SSSC/sssc24.xml (hunk 5 above)
Comment (aaronfay) [inline source/SSSC/sssc24.xml]:
```suggestion
<w canon="kâ-oh-pê-wîkimiyan" class="vta" gcat="2s1sTAcnj" preverbs="kâ|cnj ôh|thence pê|thence" sgloss="live_with" stem="wîkim" suffix="-iyan">k-ôh-pê-wîkimiyan</w>
```

Comment (aaronfay) [reply source/SSSC/sssc24.xml]: I'll do a search for these next.

File: source/SSSC/sssc26.xml (hunk 7 above)
Comment (aaronfay) [inline source/SSSC/sssc26.xml]:
```suggestion
<w canon="ê-manitowakêyimoyan" class="vai" gcat="2sAIcnj" preverbs="ê|cnj" sgloss="think_self_powered" stem="manitôwakêyimo" suffix="-yan">êh-manitôwakêyimoyin</w>
```

File: source/SSSC/sssc27.xml [hunk 15]
```diff
@@ -710,7 +710,7 @@
...
+<w canon="kâ-ôh-kî-wanihiyan" class="vta" gcat="2s1sTAcnj" preverbs="kâ|cnj ôh|thence kî|PAST" sgloss="lose" stem="wanih" suffix="-iyan">k-ô-kih-wanihiyin</w>
```
Comment (aaronfay) [inline source/SSSC/sssc27.xml]:
```suggestion
<w canon="kâ-oh-kî-wanihiyan" class="vta" gcat="2s1sTAcnj" preverbs="kâ|cnj ôh|thence kî|PAST" sgloss="lose" stem="wanih" suffix="-iyan">k-ô-kih-wanihiyin</w>
```

File: source/SSSC/sssc30.xml [hunk 16]
```diff
@@ -3875,7 +3875,7 @@
...
+<w canon="ê-pakosihtâyan" class="vai" gcat="2sAIcnj" preverbs="ê|cnj" sgloss="beg_for" stem="pakosihtâ" suffix="-yan">ê-pakosîhtâyan</w>
```
Comment (aaronfay) [inline source/SSSC/sssc30.xml]:
```suggestion
<w canon="ê-pakosîhtâyan" class="vai" gcat="2sAIcnj" preverbs="ê|cnj" sgloss="beg_for" stem="pakosihtâ" suffix="-yan">ê-pakosîhtâyan</w>
```

File: source/SSSC/sssc34.xml (hunk 11 above)
Comment (aaronfay) [reply source/SSSC/sssc34.xml:4209]: Good call, I think it makes sense in context. I'll address this in a different PR

**Pull request merged**